### ISSUE TITLE
apigee lint failing if targets/default.xml has HealthMonitor

### ISSUE THREAD

**Issue by satishbkodali** [apigee/apigeelint#320]
apigeelint failing if proxy is doing loadbalancing and health montior is failing. This is happening in later

![image](https://user-images.githubusercontent.com/22226388/184017722-8f71bbaa-0081-483c-bba9-c8196e8f9e44.png)

sample target.xml

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<TargetEndpoint name="default">
    <Description/>
    <FaultRules/>
    <PreFlow name="PreFlow">
        <Request/>
        <Response/>
    </PreFlow>
    <PostFlow name="PostFlow">
        <Request/>
        <Response/>
    </PostFlow>
    <Flows/>
     <HTTPTargetConnection>
        <LoadBalancer>
            <Algorithm>RoundRobin</Algorithm>
            <Server name="target-east"/>
            <Server name="target-south"/>
            <MaxFailures>2</MaxFailures>
            <ServerUnhealthyResponse>
            	<ResponseCode>502</ResponseCode>
        	</ServerUnhealthyResponse>
        </LoadBalancer>
        <Properties>
            <Property name="connect.timeout.millis">120000</Property>
        </Properties>
        <Path>/basepath</Path>
        <HealthMonitor>
            <IsEnabled>true</IsEnabled>
            <IntervalInSec>5</IntervalInSec>
            <HTTPMonitor>
                <Request>
                    <ConnectTimeoutInSec>10</ConnectTimeoutInSec>
                    <SocketReadTimeoutInSec>30</SocketReadTimeoutInSec>
                    <Port>443</Port>
                    <Verb>GET</Verb>
                    <Path>/basepath/health</Path>
                </Request>
                <SuccessResponse>
                    <ResponseCode>200</ResponseCode>
                </SuccessResponse>
            </HTTPMonitor>
        </HealthMonitor>
    </HTTPTargetConnection>
</TargetEndpoint>
```

@ssvaidyanathan @DinoChiesa

**Comment by ssvaidyanathan** [comment]
@satishbkodali - please provide the version you are using

**Comment by satishbkodali** [comment]
@ssvaidyanathan 2.22.0

**Comment by ssvaidyanathan** [comment]
Fixed and released in v2.23.0
